dev-manuals/services/overworld/database-model-and-type.md

```diff
@@ -15,7 +15,7 @@
    - game
    - configurationId
     
-also check [Class-Diagram](class-diagram/overworld-class-diagram.png) for this
+![Overworld Class Diagram](class-diagram/overworld-class-diagram.png)
```

Comment:
I don't understand, why doesn't GitHub display the image directly?
That should be valid markdown to insert an image...